migration/util.go

```diff
@@ -87,6 +87,10 @@ type Config struct {
 	ProgressLogPeriod time.Duration
 	// The epoch at which the upgrade will run.
 	UpgradeEpoch abi.ChainEpoch
+
+	// FIP-0081 constants for the power actor state for pledge calculations.
+	PowerRampStartEpoch     int64  // Epoch at which the new pledge calculation starts.
```

Comment:
Same comment from lotus PR -- please file an issue to clean this up after nv24

Reply:
I decided to move these to function arguments since we can't really remove them as we need the migration to stay in place into the future, so these would get stuck in this struct.